spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -641,9 +726,17 @@ def getCorpCACertSh():
 echo
 if [ $USING_SSL -eq 1 ] ; then
 
-    CERT_DIR=/usr/share/rhn
-    TRUST_DIR=/etc/pki/ca-trust/source/anchors
-    UPDATE_TRUST_CMD="/usr/bin/update-ca-trust extract"
+    if [ "$INSTALLER" == "apt" ]; then
+      CERT_DIR=/usr/local/share/ca-certificates/susemanager
+      TRUST_DIR=/usr/local/share/ca-certificates/susemanager
+      UPDATE_TRUST_CMD="/usr/bin/update-ca-trust"
```

Comment:
this is the same path as for Red Hat. Please check Line 786 if that block applies also for Ubuntu.
If yes, we need to change the comment, otherwise we need to check if that block makes trouble.

Reply:
Seems this is the last thing which needs to be checked.

Reply:
Right, good catch. I've changed the command to `/usr/sbin/update-ca-certificates` which is the Debian equivalent of `update-ca-trust`.